release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Release notes
 
+## May 28, 2019
+
+A bug fix release for the Places UI. Fixed a couple issues reported by customers:
```

Comment:
Please reword to read as follows:

"Fixed the following issues in the Places UI:

* Updated to the Solution Switcher to align with the rest of the Experience Cloud.
* Fixed an issue where the rank was saving in instances where no rank changes were made.
* Increased(?) the minimum allowed radius in the UI to 10 meters.
* Fixed an issue where, if you delete all the numbers in the field, the radius field reset back to 20 meters."